blosc/blosc2.h

```diff
@@ -250,6 +250,34 @@ enum {
     BLOSC2_VALUE_RUNLEN = 0x3,    //!< generic value run-length
 };
 
+/**
+ * @brief Error codes
+ */
+enum {
+  BLOSC2_ERROR_SUCCESS = 0,           //<! Success
```

Comment:
Maybe it would be better to rename `BLOSC2_ERROR_SUCCESS`-> `BLOSC2_SUCCESS` ?

Reply:
I could go either way. If you want to change it I'm ok or we can change it in the future.